tests/compile.test.js

```diff
@@ -99,4 +99,31 @@ describe('compile', () => {
       return null;
     });
   });
+
+  it('should use correct stats option', () => {
+    const testWebpackConfig = {
+      stats: 'minimal'
+    };
+    let mockStats = {
+      compilation: {
+        errors: [],
+        compiler: {
+          outputPath: 'statsMock-outputPath'
+        }
+      },
+      toString: sandbox.stub().returns('testStats')
+    };
+
+    module.webpackConfig = testWebpackConfig;
+    webpackMock.compilerMock.run.reset();
+    webpackMock.compilerMock.run.yields(null, mockStats);
+    return (expect(module.compile()).to.be.fulfilled)
+    .then(() => {
+      module.webpackConfig = [testWebpackConfig];
+      return (expect(module.compile()).to.be.fulfilled);
+    })
+    .then(() => {
+      expect(mockStats.toString.args).to.eql([[testWebpackConfig.stats], [testWebpackConfig.stats]]);
```

Comment:
You can do the same for "mockStats.toString.firstCall.args" to check that the plain webpack config's stats was used there.

Reply:
It already covered in that checks since sinon.args will list the history of all args passed to the spy function, but I have added the first call check. Not sure if excessive test is good or bad :smiling_imp: